frontend/javascripts/oxalis/view/layouting/golden_layout_adapter.js

```diff
@@ -103,6 +104,32 @@ export class GoldenLayoutAdapter extends React.PureComponent<Props<*>, *> {
     if (onLayoutChange != null && this.gl.isInitialised) {
       onLayoutChange(this.gl.toConfig(), this.props.activeLayoutName);
     }
+    // If the maximized item changed adjust css classes to not show hidden gl items.
+    if (this.gl && this.maximizedItem !== !this.gl._maximisedItem) {
+      // Gl needs a forced updated when returning from maximized viewing
+      // mode to render stacked components correctly.
+      const needsUpdatedSize = this.gl._maximisedItem === null && this.maximizedItem != null;
+      this.maximizedItem = this.gl._maximisedItem;
+      const allGlHeaderElemets = document.getElementsByClassName("lm_item");
+      for (const element of allGlHeaderElemets) {
+        if (this.maximizedItem) {
+          // Show only the maximized item and do not hide the gl root component.
+          if (
+            !element.classList.contains("lm_maximised") &&
+            !element.classList.contains("lm_root")
+          ) {
+            element.classList.add("hidden-gl-item");
+          }
+        } else {
+          // If there is no maximized component, remove the css class to show everything as usual.
+          element.classList.remove("hidden-gl-item");
```

Comment:
If a tab is hidden, because it is not the active tab, this does not break anything, right?

Reply:
No this does not break anything, as `hidden-gl-item` is only added to the gl-container (`gl_item`). This causes the header of that container and all its components to not being displayed. So this does not affect the hidden tabs directly.
Even if the maximized component has multiple tabs, `hidden-gl-item` is not applied and gl takes care that the hidden tabs are not displayed.

=> I could not find any problem with this. :slightly_smiling_face: 
Also removing a non existend class does not case an error.